env/main.tf

```diff
@@ -337,20 +337,20 @@ resource "azurerm_api_management_api_policy" "patient_policy" {
   <inbound>
     <base />
     <!-- Look for func-host-key in the cache -->
-    <cache-lookup-value key="func-host-key" variable-name="funchostkey" />
+    <cache-lookup-value key="${data.azurerm_key_vault_secret.fa_patient_api_host_key.version}" variable-name="funchostkey" />
```

Comment:
we should probably prefix it with "func-host-key-" to prevent clashes.

Reply:
will fix it soon